src/components/ErrorModal.vue

```diff
@@ -0,0 +1,267 @@
+        <p class="successDescription">
+          {{ $t('reportBug.Success.Description') }}
+        </p>
+        <Button
+          @click="resolve"
+        >
+          {{ $t('reportBug.Success.Return') }}
+        </Button>
+      </template>
+    </div>
+  </div>
+</template>
+
+<script>
+import { detect } from 'detect-browser';
+import FailureIcon from '../assets/APIError.svg?icon-component';
+import IconClose from '../assets/iconCloseRebranded.svg?icon-component';
+import IconEye from '../assets/iconEye.svg?icon-component';
+import IconOK from '../assets/iconOK.svg?icon-component';
+import ButtonPlain from './ButtonPlain.vue';
+import { issueReport } from '../utils/backend';
+import Button from './Button.vue';
+
+export default {
+  components: {
+    FailureIcon,
+    IconClose,
+    IconEye,
+    IconOK,
+    ButtonPlain,
+    Button,
+  },
+  props: {
+    resolve: { type: Function, required: true },
+    error: { type: [Object, Error, PromiseRejectionEvent.reason], required: true },
+  },
+  data() {
+    return {
+      report: '',
+      step: 1,
+      details: false,
+    };
+  },
+  methods: {
+    async send() {
+      const report = {
+        appVersion: process.env.npm_package_version,
+        browser: detect(),
+        error: this.error,
+        platform: 'superhero-ui',
+        description: this.report,
+        time: Date.now(),
+      };
+
+      await issueReport(report);
+
+      this.step = 2;
+    },
+    toggleDetails() {
+      this.details = !this.details;
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.error-modal {
+  padding-top: 5rem;
+  position: fixed;
+  justify-content: center;
+  align-items: center;
+  z-index: 9998;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  background-color: rgba(0, 0, 0, 0.5);
+
+  .modal-body {
+    text-align: center;
+    margin: 0 auto;
+    padding: 2.5rem 1.6em 2.5rem 1.6em;
+    background: #131313;
+    border-radius: 0.25rem;
+    border: 1px solid $card_border_color;
+    box-shadow: 4px 4px 8px rgba(0, 0, 0, 0.25);
+    height: auto;
+    max-width: 448px;
+
+    @include smallest {
+      width: 100%;
+      padding: 1rem;
+    }
+
+    img {
+      height: 2rem;
+      margin-bottom: 0.85rem;
+    }
+
+    h1 {
+      color: $standard_font_color;
+      font-size: 1rem;
+      font-weight: 500;
+    }
+
+    h2 {
+      color: #fff;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 15px;
+      line-height: 24px;
+    }
+
+    .description {
+      font-size: 15px;
+      line-height: 24px;
+      color: $tip_note_color;
+    }
+
+    .reportTitle {
+      font-size: 15px;
+      color: #787878;
+      text-align: left;
+    }
+
+    .button {
+      background-color: $secondary_color;
+      border: none;
+      border-radius: 0.25rem;
+      font-size: 0.75rem;
+      font-weight: 700;
+      justify-self: center;
+      line-height: 1.125;
+      padding: 0.65rem 1rem;
+      color: #fff;
```

Comment:
Isn't these tyles already defined in Button component?

Reply:
done